src/screens/CreatePost/elements/CreatePollContainer/CreatePollContainer.ios.js

```diff
@@ -112,7 +129,7 @@
                   }}
                   selectedValue={pickerDay}>
                   {days.map((day, index) => (
-                    <Picker.Item key={index} label={day} value={day} />
+                    <Picker.Item key={index} label={day} value={day} color={COLORS.white} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `color` prop of the `Picker.Item` component is being set to `COLORS.white`. This is a good practice as it ensures consistency in the UI design.



<!-- This is an auto-generated comment by OSS CodeRabbit -->